db/schema.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS usersData;
+
+CREATE TABLE usersData(
+   id        INTEGER  NOT NULL PRIMARY KEY 
+  ,age       INTEGER 
+  ,name      VARCHAR(15)
+  ,company   VARCHAR(9)
+  ,country   VARCHAR(11)
+  ,photo     text
+  ,about     VARCHAR(483)
+  ,latitude  NUMERIC(10,6)
+  ,longitude NUMERIC(11,6)
```

Comment:
These limits on length are too restrictive. For example, for name, `varchar(15)` means that no user can have a full name longer than 15 characters (including space). That's too short. It's possible that it works for the current data set but in real life this would cause errors for many people's names. Same thing for company and country. It's much better to give them all the same generous limit, which is why I recommend `varchar(255)`.

Similar point about latitude and longitude. Better to make this a type like `float`, `decimal`, or `double` and not worry about the exact length of the number or the number of decimal places you want.